src/pkg/variables/types.go

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package variables contains functions for interacting with variables
+package variables
+
+// VariableType represents a type of a Zarf package variable
+type VariableType string
+
+const (
+	// RawVariableType is the default type for a Zarf package variable
+	RawVariableType VariableType = "raw"
+	// FileVariableType is a type for a Zarf package variable that loads its contents from a file
+	FileVariableType VariableType = "file"
+)
```

Comment:
There is a duplicate block of this in src/types/runtime that needs to be deleted

Reply:
fixed